layout.link

```diff
@@ -244,6 +244,7 @@ WRAMX $2
 	align 8
 	"Sound Stack"
 	"Pic Animations RAM"
+	"OAM Backup"
```

Comment:
Does this need an `align 8`?

Reply:
It does not need to be aligned, as we are only copying data to the backup location and restoring afterwards.